### PR TITLE
Fixed issue : unable to update from pre 495 version

### PR DESCRIPTION
**Pull request by Shnoulle**

### CODE REVIEW

File: application/models/User.php [hunk 1]
```diff
@@ -130,6 +130,12 @@ public function rules()
     /** @inheritdoc */
     public function scopes()
     {
+        if (App()->getConfig("DBVersion") < 495) {
+            /* No expires column before 495 */
+            return array(
+                'active' => []
+            );
+        }
```
Comment (ptelu) [inline application/models/User.php:138]: you think this will be enough? what about future updates changing the model/database structure

Comment (Shnoulle) [reply application/models/User.php:138]: I really don't understand ?

`active` scopes use `expires` column. Not any other column … Why care of future column. If we need to add a scope in future we update this part

Else : i upgrade from 3 connected and not connected

Comment (olleharstedt) [reply application/models/User.php:138]: It's also a question of the Yii database schema cache and the attributes that are fetched automatically from the database. I dunno honestly.

Comment (Shnoulle) [reply application/models/User.php:138]: Can you test ?

1. At start of the update : test is done with empty active array: OK, user exist
2. Update done
3. database schema cache reset
4. test is done with real active array: OK, user exist

Potential issue if DB broke during upgrade ?

Comment (Shnoulle) [reply application/models/User.php:138]: > Potential issue if DB broke during upgrade ?

Seems not an issue :  https://github.com/LimeSurvey/LimeSurvey/blob/92806d663b4fdad285aa14488e58918091feda51/application/helpers/update/updatedb_helper.php#L97

**Pull request merged**